### PR TITLE
When loading a manifest file, check all headers are present.

### PR DESCRIPTION
**Pull request by vext01**
A small change to make the manifest parser more robust.

OK?

### CODE REVIEW

File: krun/scheduler.py [hunk 1]
```diff
@@ -84,6 +86,7 @@ def _parse(self):
         self._reset()
         fh = self._open()
         offset = 0
+        seen_headers = []
```
Comment (ltratt) [inline krun/scheduler.py]: Why not make this a set?

Comment (ltratt) [inline krun/scheduler.py]: Yes, but that's less efficient (and less obvious). Might as well bite the bullet upfront!

Comment (vext01) [inline krun/scheduler.py]: There you go.

**Comment by ltratt** [comment]
Perfect! Let's squash and merge.

**Comment by vext01** [comment]
splat.

**Pull request merged**